docs/HowToChooseTuner.md

```diff
@@ -171,6 +171,31 @@ _Usage_:
       eta: 3
 ```
 
+<a name="NetworkMorphism"></a>
+**Network Morphism**
+
+[Network Morphism](7) provides functions to automatically search for architecture of deep learning models. Every child network inherits the knowledge from its parent network and morphs into diverse types of networks, including changes of depth, width and skip-connection. Then it finds the most promising one to get the final result. More detail can be referred to [here](../src/sdk/pynni/nni/networkmorphism_tuner/README.md).
+
+_Suggested scenario_: It is suggested that you want to apply deep learning methods to your task but you have no idea of how to choose or design a network. It is feasible for different tasks to find a good network architecture.

```

Comment:
I think we should also mention that, for now, we only support cv, and we could tell the user the operation we supported now.

Reply:
The operations I mentioned above are changes of depth, width and skip-connection which we have supported yet. The cv domain I mentioned in the README.md of tuner.